Formula/ghc.rb

```diff
@@ -15,6 +15,10 @@ class Ghc < Formula
   deprecated_option "tests" => "with-test"
   deprecated_option "with-tests" => "with-test"
 
+  # https://www.haskell.org/ghc/download_ghc_8_0_1#macosx_x86_64
+  # "This is a distribution for Mac OS X, 10.7 or later."
```

Comment:
I'm not sure we need the comment and link. Are we going to update the link every release since that refers to 8_0_1 specific release notes?